src/main/resources/application-worker.yaml

```
@@ -35,3 +35,15 @@ rhsm-subscriptions:
     back-off-max-interval: ${TALLY_SUMMARY_PRODUCER_BACK_OFF_MAX_INTERVAL:1m}
     back-off-multiplier: ${TALLY_SUMMARY_PRODUCER_BACK_OFF_MULTIPLIER:2}
     max-attempts: ${TALLY_SUMMARY_PRODUCER_MAX_ATTEMPTS:5}
+  billing-producer:
+    back-off-initial-interval: ${BILLING_PRODUCER_BACK_OFF_INITIAL_INTERVAL:1s}
+    back-off-max-interval: ${BILLING_PRODUCER_BACK_OFF_MAX_INTERVAL:1m}
+    back-off-multiplier: ${BILLING_PRODUCER_BACK_OFF_MULTIPLIER:2}
+    max-attempts: ${BILLING_PRODUCER_MAX_ATTEMPTS:1}
+    incoming:
+        topic: platform.rhsm-subscriptions.tally
+        kafka-group-id: swatch-producer-billing
+        seek-override-end: ${KAFKA_SEEK_OVERRIDE_END:false}
+        seek-override-timestamp: ${KAFKA_SEEK_OVERRIDE_TIMESTAMP:}
```

nitpick: too many spaces; please use 2 less:
```suggestion
      topic: platform.rhsm-subscriptions.tally
      kafka-group-id: swatch-producer-billing
      seek-override-end: ${KAFKA_SEEK_OVERRIDE_END:false}
      seek-override-timestamp: ${KAFKA_SEEK_OVERRIDE_TIMESTAMP:}
```